WNPRC_EHR/src/components/AnimalInfoPane.tsx

```diff
@@ -47,108 +48,133 @@ const AnimalInfoPane: React.FunctionComponent<PaneProps> = props => {
       <div>
         <Table responsive="sm" className="animal-info-table">
           <tbody>
-            <tr>
-              <td>Id</td>
-              <td>
-                <a href={animalInfo._labkeyurl_Id}>{animalInfo.Id}</a>
-              </td>
-            </tr>
-            <tr>
-              <td>Status</td>
-              <td>
-                <a href={animalInfo._labkeyurl_calculated_status}>
-                  {animalInfo.calculated_status}
-                </a>
-              </td>
-            </tr>
-            <tr>
-              <td>Gender</td>
-              <td>
-                <a href={animalInfo._labkeyurl_genderd}>{animalInfo.gender}</a>
-              </td>
-            </tr>
-            <tr>
-              <td>Dam</td>
-              <td>
-                <a href={animalInfo._labkeyurl_dam}>{animalInfo.dam}</a>
-              </td>
-            </tr>
-            <tr>
-              <td>Active Assignments</td>
-              <td></td>
-            </tr>
-            <tr>
-              <td>Avail</td>
-              <td></td>
-            </tr>
-            <tr>
-              <td>Birth</td>
-              <td>
-                <a href={animalInfo._labkeyurl_birth}>{animalInfo.birth}</a>
-              </td>
-            </tr>
-            <tr>
-              <td>Age</td>
-              <td></td>
-            </tr>
-            <tr>
-              <td>Room</td>
-              <td>
-                <a href={animalInfo["_labkeyurl_Id/curLocation/room"]}>
-                  {animalInfo["Id/curLocation/room"]}
-                </a>
-              </td>
-            </tr>
-            <tr>
-              <td>Cage</td>
-              <td>
-                <a href={animalInfo["_labkeyurl_Id/curLocation/cage"]}>
-                  {animalInfo["Id/curLocation/cage"]}
-                </a>
-              </td>
-            </tr>
-            <tr>
-              <td>Condition</td>
-              <td>
-                <a href={animalInfo["_labkeyurl_Id/curLocation/cond"]}>
-                  {animalInfo["Id/curLocation/cond"]}
-                </a>
-              </td>
-            </tr>
-            <tr>
-              <td>Current Weight</td>
-              <td>
-                <a
-                  href={
-                    animalInfo[
-                      "_labkeyurl_Id/MostRecentWeight/MostRecentWeight"
+          <tr>
+            <td>Id</td>
+            <td>
+              <a href={animalInfo._labkeyurl_Id} target={"_blank"}>{animalInfo.Id}</a>
+            </td>
+          </tr>
+          <tr>
+            <td>Gender</td>
```

Comment:
Should we consider changing the Gender label to Sex to be more accurate? I know there are several places that gender is used, but it seems like something we should think about fixing as we come across instances of it.

Reply:
I know the behavioral services group has been pushing for this change. Not sure if there was ever a general consensus about it yet but I think it's a good idea to start updating that wherever possible.